src/ed25519.rs

```diff
@@ -129,7 +129,7 @@ impl SecretKey {
     /// A `SecretKey`.
     #[inline]
     #[allow(dead_code)]
-    fn from_bytes(bytes: &[u8]) -> SecretKey {
+    pub fn from_bytes(bytes: &[u8]) -> SecretKey {
```

Comment:
Oh, and now I noticed `SecretKey` is `pub struct SecretKey(pub [u8; 64])`. I guess `SecretKey(bytes)` works but I think I'd prefer some sort of constructor function.